rasgotransforms/rasgotransforms/transforms/filter/filter.sql

```diff
@@ -10,7 +10,7 @@ SELECT *
 FROM {{ source_table }}
 {% for filter_block in items %}
 {%- set oloop = loop -%}
-{{ 'WHERE ' if oloop.first else ' AND ' }}
+{{ 'WHERE ' if oloop.first else ' ' ~ all_or_any ~ ' ' }}
```

Comment:
Scratch any comment you've seen so far. I need to make sure it'll work for both preview filters and transform filters

Reply:
Instead of all_or_any it will be:
`filter_block['compoundBoolean']`